tasks/main.yml

```
@@ -23,3 +23,8 @@
   become_user: "{{ project_user }}"
   vars:
     ansible_ssh_pipelining: true
+
+- name: cp webpack-stats.json webpack-stats-live.json
+  command: cp webpack-stats.json webpack-stats-live.json
```

This might need the chdir like the task right about it:


```suggestion
  command: cp webpack-stats.json webpack-stats-live.json
  args:
    chdir: "{{ source_dir }}"
```